src/measurements.ts

```diff
@@ -140,45 +167,145 @@ export class Measurements {
     }).join("");
 
     measurementsContainer.innerHTML = rows;
-    this.addEventListeners();
+    
+    this.addEventListenersOnRemoveLineButton();
+    if (!this.listenersAdded) {
```

Comment:
```suggestion
    if (!this.eventListenersAdded) {
```

or eventListenersInitialised